capsules/src/process_console.rs

```diff
@@ -5,7 +5,7 @@
 //! --------
 //!
 //! This module provides a simple text-based console to inspect and control
-//! which processes are running. The console has five commands:
+//! which processes are running. The console has eleven commands:
```

Comment:
It's probably unlikely we'll be able to keep these comments in sync with the new document. I would rather just remove the duplicates from here and instead direct readers to the new doc file.